portal/server/controllers/start/before-you-start-bank-deal.controller.js

```diff
@@ -0,0 +1,38 @@
+import { constructPayload, errorHref, postToApi, requestParams } from '../../helpers';
```

Comment:
I am not adding tests as this is only moving code around -- and theres a lot to do!